config/team_mapping.yaml

```diff
@@ -0,0 +1,64 @@
+Baltimore Ravens:
+  - BAL
+Tennessee Titans:
+  - TEN
+New York Jets:
+  - NYJ
+Miami Dolphins:
+  - MIA
+New England Patriots:
+  - NE
+Atlanta Falcons:
+  - ATL
+San Francisco 49ers:
+  - SF
+Arizona Cardinals:
+  - ARI
+Jacksonville Jaguars:
+  - JAC
+Kansas City Chiefs:
+  - KC
+Las Vegas Raiders:
+  - LV
+Detroit Lions:
+  - DET
+Minnesota Vikings:
+  - MIN
+Chicago Bears:
+  - CHI
+Dallas Cowboys:
+  - DAL
+Seattle Seahawks:
+  - SEA
+Cincinnati Bengals:
+  - CIN
+Cleveland Browns:
+  - CLE
+Houston Texans:
+  - HOU
+Denver Broncos:
+  - DEN
+Green Bay Packers:
+  - GB
+New Orleans Saints:
+  - "NO"
+Philadelphia Eagles:
+  - PHI
+New York Giants:
+  - NYG
+Pittsburgh Steelers:
+  - PIT
+Indianapolis Colts:
+  - IND
+Buffalo Bills:
+  - BUF
+Los Angeles Chargers:
+  - LAC
+Carolina Panthers:
+  - CAR
+Tampa Bay Buccaneers:
+  - TB
+Washington:
+  - WFT
```

Comment:
Not sure if you tested this particular one / maybe you changed the source, but when I was testing the team mappings in my branch, Washington needed to be `WSH` to match with ESPN.

Reply:
Thanks for catching this. It was WFT in the schedule grid I pulled. Updated schedule grid and weekly matchups to WSH to avoid any issues later on. Renamed my team mapping to team mapping jack, since I noticed you have the same name